JS-Assignment/assignment.js

```diff
@@ -0,0 +1,340 @@
+// Create a function that takes two numbers as arguments and returns their sum.
+// addition(3, 2) ➞ 5
+// addition(-3, -6) ➞ -9
+// addition(7, 3) ➞ 10
+
+function addition(n1, n2) {
+    const sum = n1 + n2;
+    console.log(`Sum of ${n1} and ${n2} is ${sum}`);
+}
+
+addition(3, 2)
+addition(-3, -6)
+addition(7, 3)
+
+
+
+// Create a function that takes voltage and current and returns the calculated power
+// circuitPower(230, 10) ➞ 2300
+// circuitPower(110, 3) ➞ 330
+// circuitPower(480, 20) ➞ 9600
+
+function circuitPower (voltage, current) {
+    const power = voltage * current
+    console.log(`Circuit Power is : ${power}`)
+}
+
+circuitPower(230, 10)
+circuitPower(110, 3)
+circuitPower(480, 20)
+
+
+
+
+// Write a function that takes an integer minutes and converts it to seconds.
+// convert(5) ➞ 300
+// convert(3) ➞ 180
+// convert(2) ➞ 120
+
+function convert (minutes) {
+    const seconds = minutes * 60
+    console.log(`Seconds : ${seconds}`)
+}
+
+convert(5)
+convert(3) 
+convert(2) 
+
+
+
+
+// Write a function that takes the base and height of a triangle and return its area.
+// triArea(3, 2) ➞ 3
+// triArea(7, 4) ➞ 14
+// triArea(10, 10) ➞ 50
+
+function triArea (base, height){
+    const area = (1/2) * base * height
+    console.log(`Area of Triangle is : ${area}`)
+}
+
+triArea(3, 2)
+triArea(7, 4)
+triArea(10, 10)
+
+
+
+
+// Create a function that takes the age in years and returns the age in days.
+//  calcAge(65) ➞ 23725
+//  calcAge(0) ➞ 0
+//  calcAge(20) ➞ 7300
+
+function calcAge(years) {
+    const days = years * 365
```

Comment:
can we write condition for leap year

Reply:
Yes, We can
function calcAge(years) {
    const leapDays = parseInt(years / 4);
    console.log(leapDays)
    const days = (years * 365)+leapDays;
    console.log(`Age in days : ${days}`)
}